doc/2/api/controllers/server/capabilities/index.md

```diff
@@ -0,0 +1,74 @@
+---
+code: true
+type: page
+title: capabilities
+---
+
+# capabilities
+
+
+
+Returns the current Kuzzle capabilities.
+
+This action is accessible by all users, the purpose of this action is to inform the client what kind of limits or behavior they should expect from Kuzzle.
+
+---
+
+## Query Syntax
+
+### HTTP
+
+<SinceBadge version="auto-version"/>
```

Comment:
The badge should be at the beginning of the page when the entire page is new, just after the title